template.yaml

```diff
@@ -224,9 +227,17 @@ Resources:
       Timeout: 30
       MemorySize: 1536
       Role: !GetAtt NvaNviRole.Arn
+      AutoPublishAlias: live
+      SnapStart:
+        ApplyOn: PublishedVersions
       Environment:
         Variables:
           EXPANDED_RESOURCES_BUCKET: !Ref ResourcesBucket
+      Events:
+        SqsEvent:
+          Type: SQS
+          Properties:
+            Queue: !GetAtt NewCandidateQueue.Arn
```

Comment:
Isn't UpsertNviCandidateHandler the consumer of this queue?

Reply:
Thats true, this one will listen to DynamoDb event, it is implemented in other pr